### ISSUE TITLE
Cells[] return wrong Values for Column 2 and Column 3 when Row >65536

### ISSUE THREAD

**Issue by Eric1004** [EPPlusSoftware/EPPlus#234]
Hi everyone,

I occurred a strange behaviour when looping through a large excel file that I loaded an ExcelWorksheet from.

What happens:
```
var timestampAddress = (new ExcelAddress(row, 1, row, 1)).Address;
var valueAddress = (new ExcelAddress(row, column, row, column)).Address;
```
when I reach row number '65537' I get the strange behaviour where:
- worksheet.Cells["A65537"].Value returns the correct value of ["A65537"]
- worksheet.Cells["B65537"].Value returns the value of ["B1"]
- worksheet.Cells["C65537"].Value returns the value of ["C1"]
- worksheet.Cells["D65537"].Value returns the correct value of ["D65537"]
- worksheet.Cells["E65537"].Value returns the correct value of ["E65537"]
- ...continuing works for every following column...

same goes for following Rows:
- worksheet.Cells["A65538"].Value returns the correct value of ["A65538"]
- worksheet.Cells["B65538"].Value returns the value of ["B2"]
- worksheet.Cells["C65538"].Value returns the value of ["C2"]
- worksheet.Cells["D65538"].Value returns the correct value of ["D65538"]
- worksheet.Cells["E65538"].Value returns the correct value of ["E65538"]
- ...continuing works for every following column...

it doesn't matter if I access the values with Cell[row, col] or with a prepared ExcelAddress instance.

The excel file I try to process is {A1:GH171267} large. 
It's a .NET Framework 4.7 Console Application
I used EPPlus in Version 5.4.0.0 where the Error occurred and upgraded to 5.4.1.0 this morning to check if this fixes the error but it doesn't.

I hope I could clarify the issue well, I'm a bit confused myself.

Best Regards
Eric

**Comment by JanKallman** [comment]
Hello,
I can't reproduce this behavior. Can you provide a reproducable test with a workbook for this?

**Comment by Eric1004** [comment]
Hey Jan,

I uploaded a example file for this.
Simplified testcase:

```
var fi = new FileInfo(file);
using (var p = new ExcelPackage(fi))
var worksheet = p.Workbook.Worksheets["Leistung"];
```

![grafik](https://user-images.githubusercontent.com/74705537/99665810-6ab3cb80-2a6a-11eb-950c-19777b87aa06.png)

[ExcelErrorFile.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/5566675/ExcelErrorFile.xlsx)

**Comment by JanKallman** [comment]
Thanks, I'll have a look

**Comment by Eric1004** [comment]
Thanks alot!
(By the way - there are some values in Column 3 (C) much lower starting at row 70k+ - but I don't see how this should be a problem for this - or rather why)

**Comment by JanKallman** [comment]
Fix applied. If you had gaps of more than 32K rows in a column, the cell store picked the value from an incorrect page. This fix will go into version 5.4.2 released later today.

**Comment by Eric1004** [comment]
Thanks a lot for this insanely fast reply and fix Jan!

**Comment by JanKallman** [comment]
Fixed in version 5.4.2
